game/src/ecs/components/ai/AIComponent.java

```diff
@@ -34,6 +40,9 @@ public AIComponent(Entity entity, IFightAI fightAI, IIdleAI idleAI, ITransition
     }
 
     /**
+     * Creates AIComponent with default behavior. For default behavior see class documentation of
```

Comment:
Den Link in die eigene Klasse würde ich weglassen, sowas verwirrt :)

Reply:
Ich mags immer wenn in der gerenderten JavaDoc da was blaues ist 😄

Reply:
😀 ja, das ist overkill 🤣